### ISSUE TITLE
JAWS does not read individual edit or delete icon buttons in a data table row (in Windows).

### ISSUE THREAD

**Issue by lavarajallu** [carbon-design-system/carbon#13380]
<img width="518" alt="Screenshot 2023-03-21 at 4 59 33 PM" src="https://user-images.githubusercontent.com/20550556/226678590-489f9b70-ed47-4c95-8342-24f23297c52f.png">
Both icons read together in JAWS 
I checked in Firefox and Chrome in Windows 10.0
Mac is working fine as expected.

Package
carbon-components, carbon-components-react, @carbon/icons-react , @carbon/type

Browser
Chrome, Firefox  - Windows

React version
16.12.0

Console/pal
@console/pal": "1.544.1"

Description
We also use
We are trying to make it so the user can add a column above the column selected

**Comment by lavarajallu** [comment]
> @lavarajallu can you please provide a link to a page where this is happening?

<img width="1133" alt="Screenshot 2023-03-24 at 3 05 39 PM" src="https://user-images.githubusercontent.com/20550556/227482428-2ed4d90c-5eb7-42c2-a0aa-9bb14f3e5bc8.png">
@mbgower  this is the Data table in this data table we have row in this row we have export and delete icons both are read at same time together

**Comment by mbgower** [comment]
@lavarajallu Sorry, I may not have been clear. I mean, please provide a link to the implementation so that I can review your code. Thanks!
